pkg/kubelet/cm/dra/claiminfo.go

```diff
@@ -104,15 +95,14 @@ func newClaimInfo(driverName, className string, claimUID types.UID, claimName, n
 		PodUIDs:         podUIDs,
 		ResourceHandles: resourceHandles,
 	}
-	claimInfo := ClaimInfo{
+	return ClaimInfo{
 		ClaimInfoState: claimInfoState,
 		annotations:    make(map[string][]kubecontainer.Annotation),
 	}
-	return &claimInfo
 }
 
 // newClaimInfoFromResourceClaim creates a new ClaimInfo object
-func newClaimInfoFromResourceClaim(resourceClaim *resourcev1alpha2.ResourceClaim) *ClaimInfo {
+func newClaimInfoFromResourceClaim(resourceClaim *resourcev1alpha2.ResourceClaim, prepared bool, podUIDs sets.Set[string]) ClaimInfo {
```

Comment:
This API feels awkward

Reply:
Let's drop it and add a `prepared` field to the standard `newClaimInfo()` call and just call that everywhere.

Reply:
Yes, I thought about it. will do, thanks.

Reply:
done